apps/skilavottord/ws/src/app/modules/recyclingRequest/recyclingRequest.service.ts

```diff
@@ -89,11 +89,9 @@ export class RecyclingRequestService {
         )
       }
     } catch (err) {
-      throw new Error(
-        `Failed on deregistered vehicle ${vehiclePermno.slice(
-          -3,
-        )} because: ${err}`,
-      )
+      delete err.data
```

Comment:
What data are we deleting here? 
There is no data object in the error object, but there is a data in error.config, error.config.data

<img width="620" alt="image" src="https://github.com/island-is/island.is/assets/142495885/146fe9cf-d64d-4bf7-9e45-d330014f1a86">

Reply:
In one of these error logs, we are logging user/pass as the HTTP `--data` section. Updated the PR description

Reply:
I think there's also a generic error handler wrapper somewhere in `libs/shared`